### DISCUSSION TITLE
Anyone using Tibber with this?

### DISCUSSION THREAD

**Discussion by kongjudas** [ulic75/power-flow-card#66] [Q&A]
I have tried so many times to set this up, but I can't get my head around it. Has anyone set it up successfully with solar and Tibber?

**Comment by ulic75** [comment]
If you could paste a screenshot of your Tibber entities from the developer tools we might be able to help you figure it out.  Just glancing at the integration on home-assistant.io it looks like you could have what they refer to as "electricity consumption in real-time" if you have Watty or Pulse.

**Comment by kongjudas** [comment]
Sorry about the delay, but I was so into this that I forgot I had asked for help. I still haven't figured it out though. I'm using the Pulse for now, but I maybe have to look into something totally different to combine with real time solar data. Here are the Tibber entities:

<img width="872" alt="Skjermbilde 2022-07-07 kl  19 46 13" src="https://user-images.githubusercontent.com/55403016/177837045-8174059c-c12e-40fd-9303-e7509a924fb9.png">

I also have the following entities for my solar inverter (these are updated every 5 mins):

<img width="1102" alt="Skjermbilde 2022-07-07 kl  19 50 30" src="https://user-images.githubusercontent.com/55403016/177837721-67450434-a18e-432e-9189-3671d914440b.png">
:

**Reply by ulic75** [reply]
My best guesses would be:

`sensor.power_hjemme_w` is your grid consumption
`sensor.power_production_hjemme_w` is your grid production
`sensor.solis_state` or `sensor.solis_ac_output_total_power_w` is your solar output

Basically you're looking for things that report the current watts or kilowatts, which I'm assuming most of those are due to the _w suffix, but I don't know for sure.

**Reply by kongjudas** [reply]
I wasn't too far away, then. The problem seems to be that Tibber doesn't have a grid in entity separate from the combined solar/ grid consumption. This could probably be done with a template, but that always blows straight over my head.
